app/utils/DefaultTaxCodeProvider.scala

```diff
@@ -24,18 +24,18 @@ import javax.inject.{Inject, Singleton}
 @Singleton
 class DefaultTaxCodeProvider @Inject()(appConfig: AppConfig) {
 
+  private lazy val Default2022UkTaxCode           = "1257L"
   private lazy val Default2021UkTaxCode           = "1257L"
-  private lazy val Default20192020UkTaxCode       = "1250L"
+  private lazy val Default2022ScottishTaxCode     = "S1257L"
```

Comment:
Really the tax kalc library should provide this. I'll add it to the backlog

Reply:
@aerodigi version 2.2.2 of the tax kalc has a function `CalculatorUtils.defaultTaxCode().taxCode`